diff --git a/lib/index.qwik.cjs b/lib/index.qwik.cjs
index 39a76a683c88ae66d7914da026ebd3a9e91187a1..96ffc368459ccb50cb8b73284dc3494eedf5de85 100644
--- a/lib/index.qwik.cjs
+++ b/lib/index.qwik.cjs
@@ -1477,8 +1477,8 @@ function flatten(issues) {
   return flatErrors;
 }
 async function safeParseAsync(schema, input, config2) {
-  const dataset = await schema._run(
-    { typed: false, value: input },
+  const dataset = await schema['~run'](
+    { value: input },
     getGlobalConfig(config2)
   );
   return {
diff --git a/lib/index.qwik.mjs b/lib/index.qwik.mjs
index b151703c8d21ed6b82afcaabff698251e9db446b..24a389317eef927874df3925a803854a9bea06ff 100644
--- a/lib/index.qwik.mjs
+++ b/lib/index.qwik.mjs
@@ -1459,8 +1459,8 @@ function flatten(issues) {
   return flatErrors;
 }
 async function safeParseAsync(schema, input, config2) {
-  const dataset = await schema._run(
-    { typed: false, value: input },
+  const dataset = await schema['~run'](
+    { value: input },
     getGlobalConfig(config2)
   );
   return {
